packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockFLTCam.swift

```diff
@@ -2,207 +2,207 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import camera_avfoundation
+@testable import camera_avfoundation
 
 // Import Objectice-C part of the implementation when SwiftPM is used.
 #if canImport(camera_avfoundation_objc)
-  import camera_avfoundation_objc
+  @testable import camera_avfoundation_objc
 #endif
 
-final class MockFLTCam: FLTCam {
-  var setOnFrameAvailableStub: ((() -> Void) -> Void)?
-  var setDartApiStub: ((FCPCameraEventApi) -> Void)?
-  var setFocusModeStub: ((FCPPlatformFocusMode) -> Void)?
-  var getMinimumAvailableZoomFactorStub: (() -> CGFloat)?
-  var getMaximumAvailableZoomFactorStub: (() -> CGFloat)?
+final class MockFLTCam: NSObject, FLTCam {
+  var setDartApiStub: ((FCPCameraEventApi?) -> Void)?
+  var setOnFrameAvailableStub: (((() -> Void)?) -> Void)?
```

Comment:
are these just re-ordered?

Reply:
Yes aside from some very minor renames e.g. `lockCaptureStub` -> `lockCaptureOrientationStub` because this method is finally is called properly in `FLTCam`